### ISSUE TITLE
progressive saving for angular-meteor

### ISSUE THREAD

**Issue by cyrus-za** [GroundMeteor/db#130]
Check Uri's comment in https://github.com/Urigo/angular-meteor/issues/840#issuecomment-154706120

**Comment by raix** [comment]
So saving the form content temporary until the user completes wants to commit the data?

**Comment by cyrus-za** [comment]
yeah pretty much. Check out the video from John Papa - https://youtu.be/JLij19xbefI

**Comment by Urigo** [comment]
I don't think it shouldn't be related to Angular Meteor alone but as a Meteor feature.
Definitely with the new [angular Meteor 1.3 syntax](http://info.meteor.com/blog/angular-meteor-1.3)

**Comment by raix** [comment]
It's something that I'm working on in the new ground db - I'll try to get something out for you guys to test within the next 2 weeks. I'm trying to decouple the new ground db as much possible eg. allowing both ddp and http as source + potentially decouple meteor *(making tests faster and easier to write + making the project more reusable in general)*

**Comment by istrau2** [comment]
Hi all,

We have an angular 1 app running breeze that I am seriously considering rewriting using the angular-meteor project. I am wondering if this is going to happen? 

To be clear: Currently, breeze allows us to create an entity based on the metadata that it reads from the server. So, it knows what a User entity is and when I say create('user) it creates a user with all of the appropriate properties. Additionally, it will track any changes to already existing users using the unit of work pattern and allow us to commit them whenever we want. Is something like this planned for the angular-meteor and if so when?
